toolkit/tools/pkg/imagecustomizerlib/extractpartitions.go

```diff
@@ -20,29 +26,62 @@ func extractPartitions(imageLoopDevice string, outDir string, basename string, p
 		return err
 	}
 
+	// Define magicNumber and frameSize for skippable frames
```

Comment:
How did you determine these values? Should they be global constants?

Reply:
These values are defined here: https://github.com/facebook/zstd/blob/release/doc/zstd_compression_format.md#skippable-frames

Reply:
Yes, they should be global constants. I will change these values to be that👍

Reply:
Added this in, resolving the conversation.